scripts/lua/modules/ts_5min_dump_utils.lua

```diff
@@ -187,7 +187,48 @@ function ts_dump.host_update_stats_rrds(when, hostname, host, ifstats, verbose)
   ts_utils.append("host:unreachable_flows", {ifid = ifstats.id, host = hostname,
 					   flows_as_client = host["unreachable_flows.as_client"],
 					   flows_as_server = host["unreachable_flows.as_server"]},
-		  when, verbose)
+      when, verbose)
+  
+  interface.select(tostring(ifstats.id))
+   
+  local host_ = interface.getHostInfo(hostname)
```

Comment:
why are you calling `getHostInfo` again? this can be detrimental for the performance. Host info is already passed to this function as a parameter named `host`:

```
function ts_dump.host_update_stats_rrds(when, hostname, host, ifstats, verbose)
```

Calling it again will cause unnecessary extra work and will put extra pressure on the CPU.

Reply:
The content of "host_" is different from the content of "host". In "host" i haven't a lot of information that i've  in "host_"

Reply:
either add what you need in `host` from C, or do these extra calculations using the custom timeseries framework provided (see the docs) so that you'll get the metrics of interest without affecting the ntopng performance for all the users.

Reply:
ok, thank you